decidim-comments/app/packs/src/decidim/comments/comments.component.js

```diff
@@ -200,17 +201,20 @@ export default class CommentsComponent {
     this._stopPolling();
 
     this.pollTimeout = setTimeout(() => {
-      $.ajax({
-        url: this.commentsUrl,
-        method: "GET",
-        contentType: "application/javascript",
-        data: {
-          "commentable_gid": this.commentableGid,
-          "root_depth": this.rootDepth,
-          order: this.order,
-          after: this.lastCommentId
-        }
-      }).done(() => this._pollComments());
+      new Promise((resolve, reject) => {
```

Comment:
I think we don't need to wrap it in a Promise here. Just pass `this._pollComments()` to the success callback.